packages/next/src/builders/build/lib/create-package-json.ts

```diff
@@ -25,6 +25,7 @@ export async function createPackageJson(
       start: 'next start',
     },
     dependencies: {
+      ...rootPackageJson.dependencies,
       next: allWorkspaceDeps.next,
```

Comment:
We can remove these deps here since they should already be in the root `package.json`. :)